public/audiosynth.js

```diff
@@ -0,0 +1,350 @@
+var Synth, AudioSynth, AudioSynthInstrument;
```

Comment:
We should fork it (https://github.com/keithwhor/audiosynth) and publish it.
So we can import it trough webpack